content/en/security/application_security/threats/exploit-prevention.md

```diff
@@ -58,6 +58,7 @@ ASM Exploit Prevention intercepts all SQL queries to determine if a user paramet
 | Local File Inclusion (LFI)         | v2.53.0          | v2.9.0           | Avail. in Q3' 24 | Avail. in Q3' 24       |
 | SQL Injection (SQLi)               | Avail. in Q3 '24 | Avail. in Q3 '24 | Avail. in Q3 '24 | Avail. in Q3 '24       |
 
+**Note:** Estimated availability of exploit prevention support for Ruby is Q4 '24.
```

Comment:
This isn't something we typically do, see the [contribution guide](https://dd.slack.com/archives/C06F1C1J2VD/p1718722050673089)

```suggestion
**Note**: Estimated availability of Exploit Prevention support for Ruby is Q4 '24.
```